Multisig/Cross-layer/Synchronization/Repeater.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  Repeater.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 05.02.21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+/// Repeats a closure until stopped explicitly, delaying every repetition with a configured `delay` time interval.
+class Repeater {
```

Comment:
can this be replaced with a system `Timer`?

Reply:
added to next things to do list